.vscode/licence.code-snippets

```diff
@@ -0,0 +1,21 @@
+{
+  "Lisence shortcut": {
+    "prefix": "lisence",
+    "body": [
+      "/*",
+      " * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient",
+      " *",
+      " * http://rtsys.informatik.uni-kiel.de/kieler",
+      " *",
+      " * Copyright 2021 by",
+      " * + Kiel University",
+      " *   + Department of Computer Science",
+      " *     + Real-Time and Embedded Systems Group",
+      " *",
+      " * This code is provided under the terms of the Eclipse Public License (EPL).",
```

Comment:
Damm. Thank god search and replace exists... It has updated 84 files. With the power of a search regex, I also updated every "Copyright ... by" to read "Copyright 2021 by", since EPL-2.0 technically does not apply to the previous years, right? The complete Header is now:

```typescript
/*
 * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
 *
 * http://rtsys.informatik.uni-kiel.de/kieler
 *
 * Copyright 2021 by
 * + Kiel University
 *   + Department of Computer Science
 *     + Real-Time and Embedded Systems Group
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License 2.0 which is available at
 * http://www.eclipse.org/legal/epl-2.0.
 *
 * SPDX-License-Identifier: EPL-2.0
 */
```

Is that the result you intended?

Reply:
Almost what I intended, only the old year numbers should have stayed as they were. EPL states that any code may just be updated to the newer version and other Eclipse projects also kept the copyright years as they were before. The copyright itself is furthermore not affected by the license under which is is made public.